### DISCUSSION TITLE
Is there any way convert the markdown to pdf?

### DISCUSSION THREAD

**Discussion by ZanyuanYang** [remarkjs/remark#1190] [Q&A] [answered]
Is there any way convert the markdown to pdf?

**Comment by ChristianMurphy** [comment, marked as answer]
Yes, there are many.
Querying "markdown to PDF" in your search engine of choice will turn up hundreds of ways that work.
Remark can be a part of that process if wanted.

What matters more is what are you converting, what transformation do you want to do on the content, and how do you want the result to be styled?
